website/docs/docs/build/metricflow-time-spine.md

```diff
@@ -56,7 +57,7 @@ models:
 ```
 </File>
 
-For an example project, refer to our [Jaffle shop](https://github.com/dbt-labs/jaffle-sl-template/blob/main/models/marts/_models.yml) example.
+For an example project, refer to our [Jaffle shop](https://github.com/dbt-labs/jaffle-sl-template/blob/main/models/marts/_models.yml) example. Note that the [`models` key](/reference/model-properties) in the time spine configuration must be placed in your `models/` directory.
```

Comment:
should we explicitly say 'not in dbt_project.yml'?

Reply:
I think this is enough since it links to the docs!